docs/pages/guides/schema-validation/syntax.mdx

```diff
@@ -64,7 +65,6 @@ root.set("hasSiblings", true);
 
 // ❌ Invalid storage updates
 root.set("name", true);
-root.set("age", 66.5); // Invalid because `age` should be a whole number
```

Comment:
Noticed this was still in here, but this is not true since we removed the `Int` type for now. We'll reintroduce it later again as a custom data validator type (when we introduce regex, numberic ranges, etc).